### PR TITLE
Auto Dependency Updates ...

### PR DESCRIPTION
**Pull request by lguzzon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,20 +5,20 @@
   "main": "index.js",
   "dependencies": {
     "concat-stream": "~1.6.0",
-    "duplexer2": "~0.0.2",
-    "escodegen": "~1.3.2",
+    "duplexer2": "0.1.4",
```
Comment (avindra) [inline package.json]: Can you keep the `~` prefix for all versions? Otherwise people pulling this in won't be able to get patch updates automatically.

**Comment by lguzzon** [comment]
Done ...

**Comment by diasdavid** [comment]
👍  maybe use https://greenkeeper.io/ for this one?

**Comment by lguzzon** [comment]
@diasdavid or https://snyk.io/ too

**Comment by goto-bus-stop** [comment]
thanks!

**Pull request merged**